mailbox/elasticsearch-v7/src/test/java/org/apache/james/mailbox/elasticsearch/v7/ElasticSearchIntegrationTest.java

```diff
@@ -339,6 +348,7 @@ void localPartShouldBeMatchedWhenHyphen() throws Exception {
 
     @Disabled("MAILBOX-401 '-' causes address matching to fail")
     @Test
+	//@Disabled("Types cannot be provided in get mapping requests, unless include_type_name is set to true.")
```

Comment:
Do we need these commented `@Disabled`?

Reply:
Removed in https://github.com/apache/james-project/pull/288/commits/467f6cbc9c2eeba393ebf611d5ba0c466ac8aaf3